CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+## Development of web-ext
+
+To get started, first install it from source as documented in [readme](https://github.com/mozilla/web-ext/blob/master/README.md).
```

Comment:
I have linked the `source`. Please see, if it's look good to you. Otherwise I can revert it.